.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+- Please make an effort to make sure your issue isn't already reported.
+- We are not accepting bug reports for PSX mode at this time. If you are interested in helping contribute to PSX mode please do so on the [forums](https://forums.pcsx2.net/index.php). Otherwise our recommendation is that you use a proper [PSX emulator](https://emulation.gametechwiki.com/index.php/PlayStation_emulators).
+- We do not accept issues relating to upscaling at this time. We are aware of the various problems with upscaling which spans many games and having hundreds of issues for the same fundamental issues isn't particularly helpful. There are several workarounds for graphical problems that come as a result of upscaling. Please try your game at native resolution before creating an issue and if your bug is the result of upscaling please use the [forums](https://forums.pcsx2.net/index.php) or [discord](https://discord.com/invite/TCz3t9k) for assistance with various upscaling workarounds. Additionally, the [Unofficial PCSX2 Wiki](https://wiki.pcsx2.net/Main_Page) often lists various fixes for upscaling issues.
+- Please make sure your system meets our [requirements](https://github.com/PCSX2/pcsx2#system-requirements) for OS version, CPU and GPU. We do not accept bug reports for unsupported operating systems. Performance issues as a result of not meeting our hardware requirements are not valid. Please read our known issues pages for [AMD](https://github.com/PCSX2/pcsx2/wiki/OpenGL-and-AMD-GPUs---All-you-need-to-know) and [Intel](https://github.com/PCSX2/pcsx2/wiki/OpenGL-and-Intel-GPUs-All-you-need-to-know) drivers.
+
+**To Reproduce**
+Steps to reproduce the behavior
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**GS Settings**
+Any non-default settings for GS. If you don't want to list them out, please provide screenshots of your configuration window (including hw hacks if enabled)
+
+**Emulation Settings**
+Any non-default core settings. If you don't want to list them out, please provide screenshots of your configuration window. Please note that the safe preset works for most games. MTVU can have some compatibility issues so please disable it before making a report. If you need to modify the settings manually because a game requires you to do so to work, please state that explicitly.
+
+**GS Window Screenshots**
+If your issue is graphical in nature or you think screenshots will help illustrate your issue, you may do that here. Otherwise you may delete this section.
+
+**System Info (please complete the following information):**
+ - PCSX2 Revision: [e.g. dev-525]
+ - OS: [e.g. Windows 10]
+ - CPU: [e.g. i5-7600]
+ - GPU: [e.g. GTX 1070]
+
+**Logs and Dumps**
+Please feel free to attach any logs, block dumps, [gsdump](https://forums.pcsx2.net/Thread-How-to-create-a-proper-GS-dump) etc here. If your problem is graphical in nature it is highly recommended that you provide a gsdump.
```

Comment:
Maybe include the mention of uploading savestates / memcard files or even? To flesh this more out a little bit. Gsdump doesn't always help on a graphical issue or what people think of a gs problem.

Reply:
If you can provide links to those things on the forums or whatever I'll include them. But I don't really want this template to be a guide.

Reply:
Fair enough, i don't seem to find anything on the forum except what's already in the PDF guides that are included with PCSX2 or some trial and error. Should be fine with this though.